### PR TITLE
Updating baskets for new challenges

### PR DESCRIPTION
**Pull request by ekosmin**
A handful of changes to baskets:
* Baskets glow when an egg is selected, and the mouse hovers to show they're clickable
* Basket labels are now centered for any text
* Basket drakes are larger
* New challenges authored, to replace old style challenges and show new layout 

Fortunately, no changes were required to place 3 baskets instead of 4- they all get centered automatically.

### CODE REVIEW

File: src/code/templates/fv-egg-sort-game.js [hunk 1]
```diff
@@ -76,8 +76,8 @@ let animationEvents = {
           animatingDrake = new BioLogica.Organism(BioLogica.Species.Drake, egg.alleles, egg.sex);
           targetBasketIndex = basketIndex;
 
-          const leftOffset = 400,
-                topOffset = 200;
+          const leftOffset = 375,
+                topOffset = 100;
```
Comment (kswenson) [inline src/code/templates/fv-egg-sort-game.js]: These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.

File: src/code/templates/fv-egg-sort-game.js [hunk 2]
```diff
@@ -91,8 +91,8 @@ let animationEvents = {
       }},
       hatchDrakeInBasket: { id: 1, complete: false, animate: function() {
         const { scale } = _this.props,
-              leftOffset = -250,
-              topOffset = -40;
+              leftOffset = -255,
+              topOffset = -140;
```
Comment (kswenson) [inline src/code/templates/fv-egg-sort-game.js]: These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.

File: src/code/templates/fv-egg-sort-game.js [hunk 3]
```diff
@@ -104,8 +104,8 @@ let animationEvents = {
       fadeDrakeAway: { id: 2, complete: false, animate: function() {
         const { baskets, scale } = _this.props,
               targetBasket = targetBasketIndex >= 0 ? baskets[targetBasketIndex] : null,
-              leftOffset = 122,
-              topOffset = 204.5;
+              leftOffset = 70.5,
+              topOffset = 77.5;
```
Comment (kswenson) [inline src/code/templates/fv-egg-sort-game.js]: These (and other offsets like them) probably need to be unscaled. You can test it by checking with the offsets depend on the scale factor.

File: src/resources/authoring/gv2.json [hunk 4]
```diff
@@ -54,7 +54,7 @@
       "targetDrakes": [{},{}],
       "userChangeableGenes": "wings, forelimbs, hindlimbs"
     },
-    "eggSortingWings": {
+    "basicEggSortingWings": {
```
Comment (kswenson) [inline src/resources/authoring/gv2.json]: Have these naming conventions been run by Trudi? I believe there's a PT story to that effect.

File: src/stylus/fablevision/fv-egg-sort-game.styl [hunk 5]
```diff
@@ -46,21 +46,28 @@
           width:inherit
           align-items: flex-start
           .basket
-            bottom: 300px
+            bottom: 235px
             right: 250px
             .basket-image
-              background: url('../resources/fablevision/layout-d/basket.png')
+              background: url('../resources/fablevision/layout-d/basket_inactive.png')
```
Comment (kswenson) [inline src/stylus/fablevision/fv-egg-sort-game.styl]: should be `background-image`

File: src/stylus/fablevision/fv-egg-sort-game.styl [hunk 6]
```diff
@@ -46,21 +46,28 @@
...
               width: 505px
               height: 503px
               background-size: contain
               position: absolute
+            &.selected .basket-image
+              background: url('../resources/fablevision/layout-d/basket.png')
```
Comment (kswenson) [inline src/stylus/fablevision/fv-egg-sort-game.styl]: should be `background-image`

File: src/code/templates/fv-egg-sort-game.js (hunk 2 above)
Comment (ekosmin) [inline src/code/templates/fv-egg-sort-game.js]: As far as I can tell, these should actually already be getting unscaled. Below, we pass in a scale factor, which goes to the `initialElementId` HOC and unscales the values. Something is definitely still off though, you saw the scaling doesn't work as expected

**Comment by ekosmin** [comment]
I made the changes you suggested in my most recent commit. I also fixed the scaling issue that was happening- turns out I was unscaling the offsets when I actually didn't want to. Once scaling was fixed, it was relatively simple to fix animations from the eggs to the baskets, so I updated the offsets and egg sizes to get that working too.

File: src/resources/authoring/gv2.json (hunk 4 above)
Comment (ekosmin) [inline src/resources/authoring/gv2.json]: They haven't...let me ping her and ask her to fill them out in the Geniverse development master

**Pull request merged**